index.js

```diff
@@ -75,27 +75,24 @@ function gulpRequireTasks (options) {
      */
     function taskNameFromPath (modulePath) {
       const relativePath = path.relative(options.path, modulePath);
-      return removeExtension(relativePath)
-        .split(path.sep).join(options.separator)
-      ;
+      const pathInfos = path.parse(relativePath);
+      var taskNameParts = []
```

Comment:
Thank you to point out this detail about `const` :blush:. I will fix all your comment asap.